WDL2CWL_example/GATK4/VariantFilteration.cwl

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env cwl-runner
```

Comment:
Did https://github.com/common-workflow-library/bio-cwl-tools/blob/release/GATK/GATK-VariantFiltration.cwl not work for you?